Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,56 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // Person 1:
+            // Gathering Input:
+            // Name
+            Console.WriteLine("Enter Name: ");
+            string nm1 = Console.ReadLine();
+
+            // Age
+            Console.WriteLine("Enter Age: ");
+            int age1 = int.Parse(Console.ReadLine());
+        
+            // Weight
+            Console.WriteLine("Enter Weight: ");
+            int weight1 = int.Parse(Console.ReadLine());
+
+            // Height
+            Console.WriteLine("Enter Height: ");
+            float height1 = float.Parse(Console.ReadLine());
+
+            // Calculating BMI
+            float bmi1 = weight1 / (height1 * height1) * 10000;
+
+            
+            // Person 2:
+            // Gathering Input:
+            // Name
+            Console.WriteLine("Enter Name: ");
+            string nm2 = Console.ReadLine();
+
+            // Age
+            Console.WriteLine("Enter Age: ");
+            int age2 = int.Parse(Console.ReadLine());
+
+            // Weight
+            Console.WriteLine("Enter Weight: ");
+            int weight2 = int.Parse(Console.ReadLine());
+
+            // Height
+            Console.WriteLine("Enter Height: ");
+            float height2 = float.Parse(Console.ReadLine());
+
+            // Calculating BMI
+            float bmi2 = weight1 / (height1 * height1) * 10000;
+
+
+            // Outputting Results:
+            // Person 1:
+            Console.WriteLine($"My name is {nm1} and I am {age1} years old, my weight is {weight1} kg, I'm {height1} meters tall and my BMI is {bmi1}");
```

Comment:
This is fine, as it was not mentioned when it should be calculated. However, you should try and consume the code as soon as you code:
For person one, output results right after reading person1 info,
For person two, output the results right after reading person2 info.